<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWidget</class>
 <widget class="QWidget" name="DeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>120</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widgetSelect
{
background-color: rgb(89,107,169);
border:1px;
border-color:transparent;
border-style:solid;
border-radius:10px;
}</string>
  </property>
  <widget class="QWidget" name="widgetSelect" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>88</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widgetSelect
{
background-color: rgb(89,107,169);
border:1px;
border-color:transparent;
border-style:solid;
border-radius:10px;
}</string>
   </property>
   <widget class="QWidget" name="widgetDeviceMain" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>90</height>
     </rect>
    </property>
    <widget class="QGifLabel" name="label_refreshing">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/png/refreshing.gif);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbDeviceName">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>20</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#lbDeviceName
{
	font-size:16px;
	color: rgb(0, 0, 0);
	font-family:Microsoft Yahei;
	/*border-image: url(:/png/device_name_bg.png);*/
}</string>
     </property>
     <property name="text">
      <string>海康</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btStatus">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    background-color: rgba(255, 255, 255, 0);
	border-image: url(:/png/link.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>94</x>
       <y>0</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Yahei</family>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>close the program</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-image: url(:/png/device_close.png);
	background-color: rgba(255, 255, 255, 0);
}

QPushButton:hover
{
	border-image: url(:/png/device_close_foucus.png);
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:pressed
{	
	border-image: url(:/png/device_close_foucus.png);
	background-color: rgba(255, 255, 255, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>lbDeviceName</zorder>
    <zorder>closeButton</zorder>
    <zorder>btStatus</zorder>
    <zorder>label_refreshing</zorder>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGifLabel</class>
   <extends>QLabel</extends>
   <header>qgiflabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
